### ISSUE TITLE
professional slide decks

### ISSUE THREAD

**Issue by loudermilk** [loudermilk/brownbag#15]
@freshforlife would you grab some of the decks you talked about - it would be useful for us to see what others are doing so we can up-grade the TCS POC game.

**Comment by loudermilk** [comment]
ravi if you can grab a cpl decks, we can review them tmrw. that's all I need from you -- do you think you can get to it by tmrw? I would like to do before Pejman's POC is due.

**Comment by freshforlife** [comment]
Yes @loudermilk, I should be able to grab a couple of decks by tomorrow.

**Comment by loudermilk** [comment]
@freshforlife thanks looking forward to this - great idea ravi.
